src/components/CaptureDetailDialog.js

```diff
@@ -114,10 +121,35 @@ function CaptureDetailDialog(props) {
    * Render the most complete capture detail we have
    */
   useEffect(() => {
-    if (cdContext.capture) {
-      setRenderCapture(cdContext.capture);
-    } else {
-      setRenderCapture(capture);
+    const current = capture || cdContext.capture;
+    if (current) {
+      setRenderCapture({
+        status:
+          current.status ||
+          (current.active && current.approved
+            ? 'approved'
+            : current.active && !current.approved
+            ? 'pending'
+            : 'rejected'),
+        id: current.id || current.uuid,
+        reference_id: current.reference_id || current.id,
+        grower_account_id: current.grower_account_id || current.planterId,
+        wallet: current.wallet || current.planterIdentifier,
+        device_identifier:
+          current.device_identifier || current.deviceIdentifier,
+        image_url: current.image_url || current.imageUrl,
+        lat: current.lat || current.latitude,
+        lon: current.lon || current.longitude,
+        age: null,
+        captureApprovalTag: current.captureApprovalTag || null,
+        morphology: current.morphology || null,
+        note: 'My Note',
+        rejectionReason: current.rejectionReason || null,
+        species_id: current.species_id || current.speciesId,
+        token_id: current.token_id || current.tokenId,
+        created_at: current.created_at || current.timeCreated,
+        updated_at: current.updated_at || current.timeUpdated,
+      });
     }
```

Comment:
Manage different keys from old and new apis by consolidating into a new object.  The consolidation matches the new api.